packages/teams-js/src/public/authentication.ts

```diff
@@ -153,20 +172,26 @@ export namespace authentication {
   }
 
   /**
-   * Requests an Azure AD token to be issued on behalf of the app. The token is acquired from the cache
-   * if it is not expired. Otherwise a request is sent to Azure AD to obtain a new token.
+   * Requests an Azure AD token to be issued on behalf of your app in an SSO flow.
+   * The token is acquired from the cache if it is not expired. Otherwise a request is sent to Azure AD to
+   * obtain a new token.
+   * This function is used to enable SSO scenarios. Please see [this page](https://learn.microsoft.com/en-us/microsoftteams/platform/tabs/how-to/authentication/tab-sso-overview)
+   * for more detail.
```

Comment:
How about:

See [Eanble SSO for tab apps](https://learn.microsoft.com/microsoftteams/platform/tabs/how-to/authentication/tab-sso-overview) for more details.